ts/parsers/elementparser.ts

```diff
@@ -56,12 +82,15 @@ class PowerpointElementParser {
 					cx: elementOffsetPosition.cx,
 					cy: elementOffsetPosition.cy
 				},
+				table: table && table.rows.length > 0 ? table : null,
```

Comment:
check if rows property is array and is set if there is a chance that it may be undefined, need to be a bit more careful because we are not using typescript so things are strictly typed.

Reply:
replaced it with loadash isEmpty